tasks/main.yml

```diff
@@ -19,5 +19,5 @@
     enabled: "{{ postgresql_service_enabled }}"
 
 # Configure PostgreSQL.
-- import_tasks: databases.yml
 - import_tasks: users.yml
+- import_tasks: databases.yml
```

Comment:
I'ld undo this change in this PR
and get the change in through (long time open) PR https://github.com/geerlingguy/ansible-role-postgresql/pull/31 , which it would be nice to get merged!